build.sbt

```diff
@@ -61,7 +81,19 @@ lazy val activityStreamImpl = project("activity-stream-impl")
   .enablePlugins(LagomJava)
   .settings(
     version := "1.0-SNAPSHOT",
-    libraryDependencies += lagomJavadslTestKit
+    dockerRepository := Some("chirper"),
+    dockerUpdateLatest := true,
+    dockerEntrypoint ++= """-Dplay.crypto.secret="${APPLICATION_SECRET:-none}" -Dplay.akka.actor-system="${AKKA_ACTOR_SYSTEM_NAME:-activityservice-v1}" -Dhttp.address="$ACTIVITYSERVICE_BIND_IP" -Dhttp.port="$ACTIVITYSERVICE_BIND_PORT" -Dakka.actor.provider=cluster -Dakka.remote.netty.tcp.hostname="$(eval "echo $AKKA_REMOTING_BIND_HOST")" -Dakka.remote.netty.tcp.port="$AKKA_REMOTING_BIND_PORT" -Dakka.cluster.seed-nodes.0="akka.tcp://${AKKA_ACTOR_SYSTEM_NAME}@${AKKA_SEED_NODE_HOST}:${AKKA_SEED_NODE_PORT}" -Dakka.io.dns.resolver=async-dns -Dakka.io.dns.async-dns.resolve-srv=true -Dakka.io.dns.async-dns.resolv-conf=on""".split(" ").toSeq,
+    dockerCommands :=
+      dockerCommands.value.flatMap {
+        case ExecCmd("ENTRYPOINT", args @ _*) => Seq(Cmd("ENTRYPOINT", args.mkString(" ")))
+        case v => Seq(v)
+      },
+    resolvers += bintrayRepo("hajile", "maven"),
+    libraryDependencies ++= Seq(
+      lagomJavadslCluster,
```

Comment:
I wonder why is `activity-stream-impl` requiring  `lagomJavadslCluter` at all. It doesn't use Lagom's persistence or pubsub. What other feature in lagom requires clustering?
I've just checked `activity-stream-impl` and it's just a [service layer](https://github.com/lagom/activator-lagom-java-chirper/blob/159fae168e3f6110c8cf32e873e05776f2bcbecc/activity-stream-impl/src/main/java/sample/chirper/activity/impl/ActivityStreamServiceImpl.java#L47-L47).